tests/sentry/hybridcloud/test_caching.py

```diff
@@ -97,7 +97,7 @@ def test_caching_function_none_value():
     cache.clear()
 
     @back_with_silo_cache(
-        base_key="my-test-key", silo_mode=SiloMode.CONTROL, t=RpcOrganizationSummary
+        base_key="my-test-key", silo_mode=SiloMode.CONTROL, t=RpcOrganizationSummary, timeout=900
```

Comment:
Do we want to back this timeout value with an option instead? Might be nice to configure this on the fly if needed.

Reply:
Using an option for the timeout could be useful. I can look into that as we add more caching to rpc requests.